src/ pages/ProductList/components/SortMenu/SortMenu.scss

```
@@ -0,0 +1,21 @@
+.soltMenu {
+  width: 75vw;
+  margin: 0 auto;
+  margin-bottom: 15px;
+  display: grid;
+
+  ul {
+    li {
```

```suggestion
    li {
```
스타일 없는 셀렉터는 쓰실 필요 없습니다

---

수정했습니다